src/docs/upgrading.dox

```diff
@@ -1,5 +1,40 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+@section version_1130 Upgrading to Version 11.3.0
+<dl>
+
+<dt>Per-file configuration "huffman_key"</dt>
+<dd>
+Huffman encoding is no longer supported for values or keys (support for keys was removed
+in a previous release).
+</dd>
+
+<dt>Connection configuration "checkpoint_cleanup"</dt>
+<dd>
+The checkpoint_cleanup configuration now uses a sub-configuration "method" to control
+how aggressively obsolete content is removed when creating checkpoints:
+"checkpoint_cleanup=[method=[none|reclaim_space]]"
+</dd>
+
+<dt>Connection configuration "verbose"</dt>
+<dd>
+Three eviction-related verbose config values ('evict', 'evict_stuck', and 'evictserver')
+are merged into the 'eviction' config.
```

Comment:
The tense here is weird, I would replace "are" with "have been".

Reply:
See [f8282ee](https://github.com/wiredtiger/wiredtiger/pull/11194/commits/f8282eeca6963f774a55f939a011acaf71258e43)